packages/backend-common/src/tokens/ServerTokenManager.test.ts

```diff
@@ -89,101 +111,97 @@ describe('ServerTokenManager', () => {
-        new ConfigReader({
+        logger,
+      });
+      const tokenManager2 = ServerTokenManager.default({
+        config: new ConfigReader({
           backend: { auth: { keys: [{ secret: 'd4e5f6' }] } },
         }),
-      );
+        logger,
+      });
 
       const { token } = await tokenManager1.getToken();
 
       await expect(tokenManager2.authenticate(token)).rejects.toThrowError(
         /invalid server token/i,
       );
     });
-
-    it('should throw for server tokens created using a noop TokenManager', async () => {
-      const noopTokenManager = ServerTokenManager.noop();
-      const tokenManager = ServerTokenManager.fromConfig(
-        new ConfigReader({
-          backend: { auth: { keys: [{ secret: 'a1b2c3' }] } },
-        }),
-      );
-
-      const { token } = await noopTokenManager.getToken();
-
-      await expect(tokenManager.authenticate(token)).rejects.toThrowError(
-        /invalid server token/i,
-      );
-    });
   });
 
-  describe('ServerTokenManager.fromConfig', () => {
-    it('should throw if backend auth configuration is missing', () => {
-      expect(() =>
-        ServerTokenManager.fromConfig(new ConfigReader({})),
-      ).toThrow();
-    });
-
-    it('should throw if no keys are included in the configuration', () => {
-      expect(() =>
-        ServerTokenManager.fromConfig(
-          new ConfigReader({
-            backend: { auth: { keys: [] } },
+  describe('default', () => {
+    describe('production', () => {
+      it('should throw if backend auth configuration is missing', () => {
+        expect(() =>
+          ServerTokenManager.default({ config: emptyConfig, logger }),
+        ).toThrow();
+      });
+
+      it('should throw if no keys are included in the configuration', () => {
+        expect(() =>
+          ServerTokenManager.default({
+            config: new ConfigReader({
+              backend: { auth: { keys: [] } },
+            }),
+            logger,
           }),
-        ),
-      ).toThrow();
-    });
-
-    it('should throw if any key is missing a secret property', () => {
-      expect(() =>
-        ServerTokenManager.fromConfig(
-          new ConfigReader({
-            backend: {
-              auth: {
-                keys: [{ secret: '123' }, {}, { secret: '789' }],
+        ).toThrow();
+      });
+
+      it('should throw if any key is missing a secret property', () => {
+        expect(() =>
+          ServerTokenManager.default({
+            config: new ConfigReader({
+              backend: {
+                auth: {
+                  keys: [{ secret: '123' }, {}, { secret: '789' }],
+                },
               },
-            },
+            }),
+            logger,
           }),
-        ),
-      ).toThrow();
+        ).toThrow();
+      });
     });
-  });
 
-  describe('ServerTokenManager.noop', () => {
-    let noopTokenManager: TokenManager;
+    describe('development', () => {
```

Comment:
perhaps `NODE_ENV === 'development'` for clarity?

```diff
-    describe('development', () => {
+    describe(`NODE_ENV === 'development'`, () => {
```

Reply:
Done.